capstone-rs/src/arch/x86.rs

```diff
@@ -105,6 +105,11 @@ impl<'a> X86InsnDetail<'a> {
         &self.0.opcode
     }
 
+    /// Instruction encoding information, e.g. displacement offset, size.
+    pub fn encoding(&self) -> cs_x86_encoding {
+        self.0.encoding
+    }
+
```

Comment:
Please also add a test for the new method below in the `test` module.

Please try to create examples where some of the offset values are zero and others are non-zero.